tools/tests-consolidated/security/database-connection.test.ts

```
@@ -0,0 +1,598 @@
+      // This should fail because logging may expose sensitive data
+      const isValidLogLevel = ['error', 'warn', 'info'].includes(logLevel);
+      const isValidLogSensitiveData = logSensitiveData === 'false';
+      
+      expect(isValidLogLevel).toBe(true);
+      expect(isValidLogSensitiveData).toBe(true);
+    });
+  });
+});
+
+// Export for use in other test files
+export { DatabaseSecurityValidator };
```

Biome (2.1.2)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)